### PR TITLE
upgrading react-native-autocomplete-input to version 5.0.0

### PR DESCRIPTION
**Pull request by joleb**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/mrlaessig/react-native-autocomplete-input/blob/f688c039f5c6b50a7fd2792bd23ef66e8768ed70/index.js#L51>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/react-native-autocomplete-input/index.d.ts [hunk 1]
```diff
@@ -78,21 +66,16 @@ export interface AutocompleteProps<T> extends TextInputProperties {
...
 
-    /**
-     * function
-     * render custom TextInput. All props passed to this function.
-     */
-    renderTextInput?(props: TextInputProperties): ReactNode;
+    keyboardShouldPersistTaps?: 'always' | 'handeld' | 'never' | boolean;
 }
 
 export default class Autocomplete<T> extends Component<AutocompleteProps<T>> {}
```
Comment (peterblazejewicz) [inline types/react-native-autocomplete-input/index.d.ts:81]: this is now wrong IMO, if using 5.0, can you double check?:
https://github.com/mrlaessig/react-native-autocomplete-input/blob/master/index.js#L5

Comment (joleb) [reply types/react-native-autocomplete-input/index.d.ts:81]: What's exactly wrong?
Taking a look at: https://github.com/mrlaessig/react-native-autocomplete-input/blob/master/index.js#L79 shows  the values of `keyboardShouldPersistTaps`

Comment (peterblazejewicz) [reply types/react-native-autocomplete-input/index.d.ts:81]: sorry to be succint on the words, the export is not a class anymore, functioinal component IMO
I didn't mean props, sry!

Comment (joleb) [reply types/react-native-autocomplete-input/index.d.ts:81]: Ah ok, you were talking about line 81. I got it and will fix it :)

**Comment by joleb** [comment]
@peterblazejewicz I fixed it now, please take a look again if possible :D and thanks for pointing the issue out.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@joleb thx!

**Comment by joleb** [comment]
Ready to merge

**Pull request merged**